src/Analyzers/CSharp/Analyzers/UsePatternCombinators/CSharpUsePatternCombinatorsDiagnosticAnalyzer.cs

```diff
@@ -18,6 +20,10 @@ namespace Microsoft.CodeAnalysis.CSharp.UsePatternCombinators
     internal sealed class CSharpUsePatternCombinatorsDiagnosticAnalyzer :
         AbstractBuiltInCodeStyleDiagnosticAnalyzer
     {
+
+        private static readonly LocalizableResourceString s_safePatternTitle = new(nameof(CSharpAnalyzersResources.Use_pattern_matching), CSharpAnalyzersResources.ResourceManager, typeof(CSharpAnalyzersResources));
+        private static readonly LocalizableResourceString s_unsafePatternTitle = new(nameof(CSharpAnalyzersResources.Use_pattern_matching_may_have_side_effects), CSharpAnalyzersResources.ResourceManager, typeof(CSharpAnalyzersResources));
```

Comment:
@CyrusNajmabadi Any concerns to always passing `CSharpAnalyzersResources.Use_pattern_matching` to the base constructor?

Reply:
fixed this up.